netbox_netprod_importer/devices_list.py

```diff
@@ -57,8 +57,9 @@ def parse_filter_yaml_def(filter_yaml, creds=None):
                 continue
 
             try:
-                if device["primary_ip"].get("address"):
-                    dev = device["primary_ip"].get("address").split("/")[0]
+                if device.get("primary_ip") and \
+                        device["primary_ip"].get("address"):
```

Comment:
You can chain it on a single line: `if device.get("primary_ip", {}).get("address"):`

Reply:
The line length is 85 characters

Reply:
```
                if device.get("primary_ip", {}).get("address"):
```
is 63 chars long. Please replace your condition and line break (I know why you did it, I just gave you a shorter version) with that.

Reply:
Sorry, I didn’t wake up in the morning :), I changed it.